variables.tf

```diff
@@ -58,6 +60,12 @@ variable "create_instance" {
   default     = true
 }
 
+variable "enable_autoscaling" {
```

Comment:
This probably bumps the change from a `fix` to a `feat`?

Reply:
Done